source/api/discovery/0.5/index.md

```diff
@@ -82,127 +82,138 @@ The key words _MUST_, _MUST NOT_, _REQUIRED_, _SHALL_, _SHALL NOT_, _SHOULD_, _S
+This specification describes three levels of conformance that build upon each other in terms of functionality and precision of the information published. Sets of changes are published in pages, which are then aggregated into a collection per publisher. To reduce barriers to entry, care has been taken to allow for the possibility of implementing all levels using only static files on a web server, rather than requiring dynamic access to a database.
 
 ### 2.1. Listing Resources and their Changes
 {: #listing-resources-and-their-changes}
 
-There are three levels of conformance at which changes can be described. Level 0 is simply a list of the resources available.  Level 1 adds timestamps and ordering from earliest change to most recent, allowing a consuming application to work backwards through the list and stop processing once it encounters a change that it has already seen from a previous run. Level 2 adds information about the types of activities, enabling the explicit description of the creation and deletion of resources.
+There are three levels of conformance at which changes can be described. Level 0 is simply a list of the resources available. Level 1 adds timestamps and ordering from earliest change to most recent, allowing a consuming application to work backwards through the list and stop processing once it encounters a change that it has already seen from a previous run. Level 2 adds information about the types of activities, enabling the explicit description of the creation and deletion of resources.
 
 The subsections below describe first how to construct the description of the changes for each level, and then in the next section how to embed them into ordered lists.
 
 #### 2.1.1. Level 0: Resource References
 {: #level-0-basic-resource-list}
 
-The core information required to provide a minimally effective set of links to IIIF resources is just the URIs of those resources. However, with the addition of some additional JSON structure wrapped around those URIs, the document can also be on the path towards a robust set of information that clients can use to optimize their processing.
-
-Starting with the IIIF resource URIs, we add an "Update" Activity wrapper around them.  The order of the resources in the resulting list is unimportant, but each should only appear once. In terms of optimization, this approach provides no additional benefit over any other simpler list format, but is compatible with the following levels which introduce significant benefits.  This is the minimum level for any effective interoperability.
+The core information required to provide a minimally effective set of links to IIIF resources is the URIs of those resources. The order of the resources in the resulting list is unimportant, but each _SHOULD_ only appear once. This is the minimum level for any effective interoperability.
```

Comment:
This read too much like a tutorial for me, and the tone wasn't consistent w/ the rest of the spec.

Reply:
Note also the addition of an RFC _SHOULD_

Reply:
A bit tutorial-like but it provides the rationale for the inclusion of the level at all, and a way to understand the approach. I think the issue is that the two paragraphs both have bits of normative spec-y text and introductory/background/rationale text.

How about your paragraph, plus:

> This approach provides no additional benefit over any simple list format, however is compatible with the following levels which introduce significant benefits. By implementing a minimal level 0 API, a provider can make a start towards later publishing a more robust and useful API that allows for clients to better optimize their interactions.

Reply:
To me it just sounds like it's trying to justify or apologize for the extra structure at level 0, or suggest that 0 isn't actually useful. What actually makes it any easier/more practical to implement? You can do it all with filesystem metadata?

Reply:
It doesn't have to be ordered, and you don't need to know the last time it was changed, just that it exists.  So yes, it is a bit of an apology, as all you're really conveying is an unordered set of URIs, which could have been a plain text file with one URI per line. It's also much less useful than level 1 or 2, as you need to check the entire list every time.  Thus both of those comments are very much warranted, but intentionally so.

Reply:
Rob's para with less apology:

> This minimal level 0 API approach is compatible with the levels 1 and 2 which introduce significant benefits that allow clients to better optimize their interactions.

Reply:
I added this in the latest push